xarray/core/accessors.py

```diff
@@ -146,3 +189,23 @@ def f(self, dtype=dtype):
     time = _tslib_field_accessor(
         "time", "Timestamps corresponding to datetimes", object
     )
+
+    def _tslib_round_accessor(self, name, freq, dtype=None):
```

Comment:
The `pandas.Series.dt.round()` and the other methods always return datetime64, so I don't think you need the `dtype` argument.

Reply:
Fixed